projects/js-packages/critical-css-gen/package.json

```
@@ -15,48 +15,45 @@
 	"license": "GPL-2.0-or-later",
 	"author": "Automattic",
 	"scripts": {
-		"build:browser": "rollup -c",
-		"build:node": "tsc",
-		"build": "pnpm run clean && pnpm run build:browser && pnpm run build:node",
-		"clean": "rm -rf build-node/ && rm -rf build-browser/",
-		"test": "pnpm build && NODE_ENV=test NODE_PATH=./node_modules jest --forceExit --config=tests/config/jest.config.js"
+		"build": "pnpm run clean && tsc",
+		"build:test": "pnpm run clean:test && webpack --config tests/data/webpack.config.cjs",
+		"clean": "rm -rf build/",
+		"clean:test": "rm -rf tests/build/",
+		"test": "pnpm build && pnpm build:test && NODE_ENV=test NODE_PATH=./node_modules jest --forceExit --config=tests/config/jest.config.js"
 	},
-	"main": "./build-node/node.js",
-	"browser": "./build-browser/bundle.js",
+	"main": "./build/browser.js",
 	"devDependencies": {
+		"@automattic/jetpack-webpack-config": "workspace:*",
 		"@babel/core": "7.24.7",
 		"@babel/preset-env": "7.24.7",
 		"@babel/preset-typescript": "7.24.7",
-		"@rollup/plugin-commonjs": "26.0.1",
-		"@rollup/plugin-json": "6.1.0",
-		"@rollup/plugin-node-resolve": "15.3.0",
-		"@rollup/plugin-terser": "0.4.3",
-		"@rollup/plugin-typescript": "12.1.0",
 		"@types/clean-css": "4.2.11",
 		"@types/css-tree": "2.3.8",
 		"@types/node": "^20.4.2",
 		"express": "4.20.0",
 		"jest": "29.7.0",
+		"path-browserify": "1.0.1",
 		"playwright": "1.45.1",
 		"playwright-core": "^1.45.1",
 		"prettier": "npm:wp-prettier@3.0.3",
-		"rollup": "3.29.5",
-		"rollup-plugin-polyfill-node": "0.13.0",
+		"process": "0.11.10",
 		"source-map": "0.7.4",
 		"source-map-js": "1.2.0",
 		"tslib": "2.5.0",
 		"typescript": "5.0.4",
 		"webpack": "5.94.0",
+		"webpack-cli": "4.9.1",
 		"webpack-dev-middleware": "5.3.4"
 	},
 	"exports": {
 		".": {
-			"jetpack:src": "./src/node.ts",
-			"types": "./build-node/node.d.ts",
-			"browser": "./build-browser/bundle.js",
-			"import": "./build-node/node.js",
-			"require": "./build-node/node.js",
-			"default": "./build-node/node.js"
+			"jetpack:src": "./src/browser.ts",
+			"types": "./build/browser.d.ts",
+			"default": "./build/browser.js"
+		},
+		"./playwright": {
+			"default": "./build/playwright.js",
+			"types": "./build/playwright.d.ts"
```

`default` should always be last. Also we may as well add a `jetpack:src` in here.
```suggestion
			"jetpack:src": "./src/playwright.ts",
			"types": "./build/playwright.d.ts",
			"default": "./build/playwright.js"
```